src/classes/OPP_CampaignMember_TDTM.cls

```diff
@@ -58,7 +58,7 @@ public class OPP_CampaignMember_TDTM extends TDTM_Runnable {
         
         dmlWrapper dmlWrapper = new DmlWrapper();
 
-        if (hasRun) {
+        if (recursionFlag > 0) {
```

Comment:
This was the example I called out in the Quip doc. My reasoning is that the static var is never referenced from outside the class, thus creating a ProcessControl class did not seem appropriate.